### ISSUE TITLE
java.lang.NoSuchFieldError: android.support.v4.app._HoloFragment.mChildFragmentManager

### ISSUE THREAD

**Issue by ajmalsalim** [Prototik/HoloEverywhere#555]
Hi,

I am trying to use ABS + HoloEverywhere + Sliding Menu. The app works perfectly when i use import android.support.v4.app.Fragment;. I need to use the PreferenceFragment and I have changed all imports to org.holoeverywhere.app.*; along by changing public class SlidingFragmentActivity extends Activity implements SlidingActivityBase. 

So all the 3 libraries are loaded and I am getting this error on rotating screen or when the HoloFragments onDestroyView() is called. Here is the error log 

java.lang.NoSuchFieldError: android.support.v4.app._HoloFragment.mChildFragmentManager
07-25 15:13:23.783: E/AndroidRuntime(475):  at android.support.v4.app._HoloFragment.onDestroyView(_HoloFragment.java:190)

07-25 15:13:23.783: E/AndroidRuntime(475):  at org.holoeverywhere.app.Fragment.onDestroyView(Fragment.java:143)
07-25 15:13:23.783: E/AndroidRuntime(475):  at android.support.v4.app.Fragment.performDestroyView(Fragment.java:1364)
07-25 15:13:23.783: E/AndroidRuntime(475):  at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:966)
07-25 15:13:23.783: E/AndroidRuntime(475):  at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1083)
07-25 15:13:23.783: E/AndroidRuntime(475):  at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1065)
07-25 15:13:23.783: E/AndroidRuntime(475):  at android.support.v4.app.FragmentManagerImpl.dispatchDestroy(FragmentManager.java:1877)
07-25 15:13:23.783: E/AndroidRuntime(475):  at android.support.v4.app.FragmentActivity.onDestroy(FragmentActivity.java:316)
07-25 15:13:23.783: E/AndroidRuntime(475):  at android.support.v4.app._HoloActivity.onDestroy(_HoloActivity.java:425)
07-25 15:13:23.783: E/AndroidRuntime(475):  at org.holoeverywhere.app.Activity.onDestroy(Activity.java:259)
07-25 15:13:23.783: E/AndroidRuntime(475):  at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:3642)
07-25 15:13:23.783: E/AndroidRuntime(475):  at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:3673)
07-25 15:13:23.783: E/AndroidRuntime(475):  at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:3789)
07-25 15:13:23.783: E/AndroidRuntime(475):  at android.app.ActivityThread.access$2400(ActivityThread.java:125)
07-25 15:13:23.783: E/AndroidRuntime(475):  at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2037)
07-25 15:13:23.783: E/AndroidRuntime(475):  at android.os.Handler.dispatchMessage(Handler.java:99)
07-25 15:13:23.783: E/AndroidRuntime(475):  at android.os.Looper.loop(Looper.java:123)
07-25 15:13:23.783: E/AndroidRuntime(475):  at android.app.ActivityThread.main(ActivityThread.java:4627)
07-25 15:13:23.783: E/AndroidRuntime(475):  at java.lang.reflect.Method.invokeNative(Native Method)
07-25 15:13:23.783: E/AndroidRuntime(475):  at java.lang.reflect.Method.invoke(Method.java:521)
07-25 15:13:23.783: E/AndroidRuntime(475):  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:868)
07-25 15:13:23.783: E/AndroidRuntime(475):  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:626)
07-25 15:13:23.783: E/AndroidRuntime(475):  at dalvik.system.NativeStart.main(Native Method)

and this is the code snippet onDestroyView()

 @Override
    public void onDestroyView() {
        super.onDestroyView();
        if (mChildFragmentManager != null && mChildFragmentManager.mActive != null
                && mDetachChildFragments) {
            for (Fragment fragment : mChildFragmentManager.mActive) {
                if (fragment == null || !fragment.mFromLayout) {
                    continue;
                }
                mChildFragmentManager.detachFragment(fragment, 0, 0);
            }
        }
    }

I have searched ans tried a lot but did not find any solution regarding this. Any help will be really appreciated!

**Comment by ajmalsalim** [comment]
I have commented the whole code inside onDestroyView() and its working now without errors. I dont know if this is a fix  or not and in future it may create issues I think.

**Comment by Prototik** [comment]
Update support library to r13.1 (r11 minimal)

**Comment by ajmalsalim** [comment]
I have tried the latest v18 and still there is issue. Also I found another problem that onclicking the actionbar home button, the slidingmenu opens without the animation and the menu is unclickable. But there is no issue if I swipe the screen to open sliding menu. I am facing this issue only when I am importing HoloFragment. 

I have added the android support library by using the option "Add support library"
